site/docs/voice/bxml/startStream.mdx

```diff
@@ -0,0 +1,340 @@
+| destination        | (required) A websocket URI to send the stream to. The audio from the specified tracks will be sent via websocket to this URL encoded as base64 encoded PCMU/G711 audio. See below for more details on the websocket packet format. |
+| streamEventUrl    | (optional) URL to send the associated Webhook events to during this stream's lifetime. Does not accept BXML. May be a relative URL.                                         |
+| streamEventMethod | (optional) The HTTP method to use for the request to `streamEventUrl`. GET or POST. Default value is POST.                                                                                                  |
+| username           | (optional) The username to send in the HTTP request to `streamEventUrl`. If specified, the URLs must be TLS-encrypted (i.e., `https`).                                           |
+| password           | (optional) The password to send in the HTTP request to `streamEventUrl`. If specified, the URLs must be TLS-encrypted (i.e., `https`).                                           |
+
+If the `streamEventsUrl` attribute is specified, then the [Media Stream Started][2], [Media Stream Rejected][3] and [Media Stream Stopped][4] events will be sent to the URL when the stream starts, if there is an error starting the stream and when the stream ends respectively. BXML returned in response to this callback will be ignored.
+
+:::note
+While multiple streams for the same call are allowed, each stream MUST have a unique name. Attempting to start a stream on the same call with the name of an already existing stream will result in a [Media Stream Rejected][3] event.
+:::
+
+## Webhooks Received
+
+| Webhooks                   | Can reply with more BXML |
+|:---------------------------|:-------------------------|
+| [Media Stream Started][2]  | No                       |
+| [Media Stream Rejected][3] | No                       |
+| [Media Stream Stopped][4]  | No                       |
+
+## Nested Tags
+
+You may specify up to 12 `<StreamParam/>` elements nested within a `<StartStream>` tag. These elements define optional user specified parameters that will be sent to the destination URL when the stream is first started.
+
+### StreamParam Attributes
+
+| Attribute | Description                                                    |
+|:----------|:---------------------------------------------------------------|
+| name      | (required) The name of this parameter, up to 256 characters.   |
+| value     | (required) The value of this parameter, up to 2048 characters. |
+
+## Websocket Packet Format
+
+At the destination end, the websocket will receive messages containing JSON for the duration of the stream. There will be an initial `start` message when the connection is first established. This will be followed by zero or more `media` messages containing the encoded audio for the tracks being streamed. Finally when a stream is stopped a `stop` message will be sent.
```

Comment:
Gimme mah commas
```suggestion
At the destination end, the websocket will receive messages containing JSON for the duration of the stream. There will be an initial `start` message when the connection is first established. This will be followed by zero or more `media` messages containing the encoded audio for the tracks being streamed. Finally, when a stream is stopped, a `stop` message will be sent.
```